core/tests/protractor_utils/workflow.js

```diff
@@ -178,7 +178,7 @@ var createAddExpDetailsAndPublishExp = async function(
 
 // Creates and publishes a exploration with two cards.
 var createAndPublishTwoCardExploration = async function(
-    title, category, objective, language) {
+    title, category, objective, language, correctnessFeedbackIsEnabled) {
```

Comment:
This parameter controls whether we enable correctness feedback, so maybe we should name it `enableCorrectnessFeedback` instead? Right now it sounds like it should describe whether correctness feedback has already been enabled